packages/core/src/shared/errors.ts

```diff
@@ -345,7 +345,7 @@ export function getTelemetryResult(error: unknown | undefined): Result {
  */
 export function scrubNames(s: string, username?: string) {
     let r = ''
-    const fileExtRe = /\.[^.\/]{1,4}$/
+    const fileExtRe = /\.[^.\/]+$/
```

Comment:
If the name is really long it's probably not an extension. Do we actually care about names >6 chars?